kubel.el

```diff
@@ -372,6 +372,15 @@ TYPENAME is the resource type/name."
     (kubel--exec buffer-name nil (list "describe" "pod" (kubel--get-pod-under-cursor)))
     (beginning-of-buffer)))
 
+(defun kubel-get-resource-details ()
+  "Get the details of the resource under the cursor."
+  (interactive)
+  (let* ((pod (kubel--get-pod-under-cursor))
```

Comment:
would it be worth renaming `kubel--get-pod-under-cursor` to something like `kubel--get-resource-under-cursor` ?

Reply:
Done! Will test for regressions.